canopy_main.ml

```diff
@@ -41,10 +41,10 @@ module Main  (C: CONSOLE) (S: STACKV4) (RES: Resolver_lwt.S) (CON: Conduit_mirag
     let module Store = Canopy_store.Store(C)(Context)(Inflator) in
     let open Canopy_utils in
     let cache = ref (KeyMap.empty) in
-    let open Canopy_config in
-    let config = config () in
+    Store.pull console >>= fun () ->
+    Store.fill_cache cache >>= fun _ ->
```

Comment:
we do `pull` and `fill_cache` here, that's fine.  can we error here if uuid is not present (I wonder when the exception is `raised`)?